code/modules/vehicles/mecha/equipment/tools/other_tools.dm

```diff
@@ -285,7 +285,7 @@
 	///Maximum fuel capacity of the generator, in units
 	var/max_fuel = 75 * SHEET_MATERIAL_AMOUNT
 	///Energy recharged per second
-	var/rechargerate = 10
+	var/rechargerate = 5 KILO WATTS
```

Comment:
What's up with this?

Reply:
Mecha plasma generator. Burns plasma to recharge Mecha cell like this 

https://github.com/tgstation/tgstation/blob/e79b3c6f928407c8d6138e9373617346e1eab821/code/modules/vehicles/mecha/equipment/tools/other_tools.dm#L346

Reply:
I halved the value because wait time is 2